README.md

```diff
@@ -96,13 +37,17 @@ docker run -p 7480:7480 ghcr.io/aquarist-labs/s3gw:latest
 For more information on building and running a container, please read our
 [guide](./build/).
 
+## Documentation
+
+You can access our documentation [here][2].
+
 ## License
 
 Licensed under the Apache License, Version 2.0 (the "License");
 you may not use licensed files except in compliance with the License.
 You may obtain a copy of the License at
 
-  <http://www.apache.org/licenses/LICENSE-2.0>
+<http://www.apache.org/licenses/LICENSE-2.0>
```

Comment:
this removes the indentation from the license notice. Unsure if it was meant, but looks weird this way :shrug:

Reply:
I need to review my linter rules 😄. I sorted it out